gfx/sfml_painter.cpp

```diff
@@ -20,10 +21,18 @@
 namespace gfx {
 namespace {
 
+auto get_font_dir_iterator(std::filesystem::path const &path) try {
+    return std::filesystem::recursive_directory_iterator(path);
+} catch (std::filesystem::filesystem_error const &) {
+    return std::filesystem::recursive_directory_iterator();
+} catch (...) {
+    throw;
+}
```

Comment:
Might be good to not silence all exceptions, only `filesystem_error`. Ok?

Reply:
Yeah, probably best to only silence filesystem_error since we know it happens and that it's harmless when it does. No need to catch and rethrow everything else since we're not doing anything with it though!

Reply:
Ah, yea, of course!